microsoft/testsuites/performance/storageperf.py

```diff
@@ -291,6 +291,11 @@ def _perf_nfs(
         for node in [server_node, client_node]:
             node.tools[Sysctl].write("fs.aio-max-nr", origin_value[node.name])
 
+        # clean up
```

Comment:
It should be in a final block to make sure it's always called. But not wrap above long content in the try/final block. create another method to put above content.

Reply:
fixed

Reply:
Please take the comment " But not wrap above long content in the try/final block. create another method to put above content."

Reply:
You mean the content in the current try block? The function `_perf_nfs` itself was created to encapsulate this.

Reply:
The try block is too long, and not clear. I double looked the code, the try can start from Ln257 `# setup nfs on server`. So it will be shorter 50%.

Reply:
Actually, we need to start it from Line250. I missed deleting `server_raid_disk_mount_dir` as part of cleanup. Let me add this to cleanup as well.